src/main/resources/view/MeetingClearConfirmWindow.fxml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this fxml file is exactly the same as `ContactsClearConfirmWindow.fxml`? Could we just reuse it as `ConfirmWindow.fxml` instead of creating a new one?